mongodbatlas/client.go

```diff
@@ -0,0 +1,90 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+	"net/http/httputil"
+	"os"
+
+	"github.com/mongodb-forks/digest"
+	"github.com/turbot/steampipe-plugin-mongodbatlas/constants"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+// getMongodbAtlasClient :: returns a mongodbatlas client to perform API requests.
+func getMongodbAtlasClient(ctx context.Context, d *plugin.QueryData) (*mongodbatlas.Client, error) {
```

Comment:
```suggestion
func getMongoDBAtlasClient(ctx context.Context, d *plugin.QueryData) (*mongodbatlas.Client, error) {
```

In other function names, I saw camel case with acronyms capitalized, so it seems good to keep that consistent